wkcuber/api/mag_view.py

```diff
@@ -91,14 +92,30 @@ def __init__(
             False,
         )
 
-        self.layer = layer
-        self.name = name
+        self._layer = layer
+        self._name = name
```

Comment:
I'd rather use the `mag` itself as the main identifier and also pass it in the init instead of the name. The name could still be available as a property using `self._mag.to_layer_name()`.

Reply:
Then also the `Mag(self.name)` could be simple `self.mag`/`self._mag` calls.